xamarin/Xamarin.Forms
# Issue4879 ImageButton Padding (UWP)

### Description of Change ###

Fix UWP so it applies `Padding` to `ImageButton` in the same way as Android and iOS.

Includes an automated test that requires `ManualReview` because unable to automate checking that Button and ImageButton render the same size visually.

### Issues Resolved ### 


- fixes #4879

### API Changes ###


- UWP

### Behavioral/Visual Changes ###


When padding is applied to an ImageButton on UWP, the padding is now applied to the button as a whole, not just the image. ImageButtons should now display the correct size and not smaller than they should be.
Incorrect behavior meant that the padding was previously applied to the nested image element.

### Before/After Screenshots ### 


In these images, the green background is on a Button containing an image. The Red and Purple images are ImageButtons, to show they now produce the same height and width as a Button containing an image.

Before:
![image](https://user-images.githubusercontent.com/189547/55684124-c1c16700-593f-11e9-99a5-112b9f1d0101.png)

After:
![image](https://user-images.githubusercontent.com/189547/55684611-dfdd9600-5944-11e9-98b3-6e87b44f5fce.png)


### Testing Procedure ###


Run the test and verify that Green, Purple, and Red "Button"s are the same size.

![Issue4879-test-run-recording](https://user-images.githubusercontent.com/189547/55684603-bcb2e680-5944-11e9-9af8-509201bc045b.gif)

### PR Checklist ###
